### ISSUE TITLE
Multiclass vector support

### ISSUE THREAD

**Issue by spatiallysaying** [opengeos/geoai#268]
### Description

Could you please share examples that work on Shapefiles/GeoJSON polygons in place of Raster Masks. There are two possibilities  1) Polygons belonging to different classes are in same Shapefile 2) Each class is in a different Shapefile.

For example, 

1. Single Geotiff Image  & A single Shapefile/GeoJSON may have swimming pool , tennis court, base ball ground,basket ball court
2. Single Geotiff Image  & Multiple Shapefiles/GeoJSON for each class -eg:- Roads, Buildings, Water Bodies etc

### Source code
In the example below 'num_classes' is clear. But, how do we associate column that has 'class name' ? 
```
out_folder = "buildings"
tiles = geoai.export_geotiff_tiles(
    in_raster=train_raster_path,
    out_folder=out_folder,
    in_class_data=train_vector_path,
    tile_size=512,
    stride=256,
    buffer_radius=0,
)

# Train U-Net model
geoai.train_segmentation_model(
    images_dir=f"{out_folder}/images",
    labels_dir=f"{out_folder}/labels",
    output_dir=f"{out_folder}/unet_models",
    architecture="unet",
    encoder_name="resnet34",
    encoder_weights="imagenet",
    num_channels=3,
    num_classes=2,  # background and building
    batch_size=8,
    num_epochs=3,
    learning_rate=0.001,
    val_split=0.2,
    verbose=True,
)
```

**Comment by giswqs** [comment]
The `in_class_data` param accepts a vector dataset with the `class` column. 

```python
export_geotiff_tiles(
    in_raster,
    out_folder,
    in_class_data,
    class_value_field="class",
```
